llms/googleai/palm/palm_llm.go

```diff
@@ -111,6 +112,9 @@ func newClient(opts ...Option) (*palmclient.PaLMClient, error) {
 	if len(options.projectID) == 0 {
 		return nil, ErrMissingProjectID
 	}
+	if len(options.location) == 0 {
+		return nil, ErrMissingLocation
+	}
 
-	return palmclient.New(options.projectID, options.clientOptions...)
+	return palmclient.New(context.TODO(), options.projectID, options.location, options.clientOptions...)
```

Comment:
There was a default location before - has it been removed?

Reply:
I felt the previous endpoint was dead and couldn't be set, possibly due to a bug, so I modified it.